Anime-App/Feature/Anime/URL/URL.swift

```diff
@@ -18,7 +19,9 @@ extension UrlEndpoint {
         case .getAnime:
             return "/anime"
         case .searchAnime:
-            return "/anime?filter[text]="
+            return "/anime?"
+        case .pagination:
+            return "?"
```

Comment:
Do you even need this case? Can't you use use the get anime case and add query params to it? Same with search. I think you should only have one case here